### DISCUSSION TITLE
Another code to print meetings list on PDF

### DISCUSSION THREAD

**Discussion by adminaa87** [code4recovery/pdf#67] [Q&A] [answered]
Hello, I am not sure if I should ask here...

I dont remember when, but maybe 2 years ago. I saw a conference about a module that generate more elaborate PDF from a TSML.

is there someone who know about it?

Thank you

**Comment by tim-rohrer** [comment, marked as answer]
There are two options I am aware of: the [Code for Recovery PDF service](https://pdf.code4recovery.org/) (actively maintained/developed), and [a PDF plugin](https://wordpress.org/plugins/12-step-meeting-pdf-generator/) developed by another individual.
